.github/workflows/embedding-sdk.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         with:
           remove-previous-comment: true
           add-new-comment: ${{ job.status == 'failure' }}
-          message: "📝 One or more internal Metabase types were accidentally exported from the `@metabase/embedding‑sdk‑react` package in this pull request.<br/><br/>Publishing these types would change the public API and may drastically change generated SDK documentation.<br/><br/>Please update the code so internal types remain private and the public API stays unchanged."
+          message: "📝 One or more new types were exported as public types from the `@metabase/embedding-sdk-react` package in this pull request.<br><br>Publishing these types may modify the public API and drastically alter the generated SDK documentation.<br><br>Please check the CI logs to find the name(s) of the exported type(s).<br><br>- If a type should be publicly available in the SDK, add a TypeDoc annotation.<br>- If a type is internal and should _not_ be publicly exposed, adjust types to avoid leaking internal types.<br><br>You can run `yarn embedding-sdk:docs:generate` locally to verify that the issue has been resolved."
```

Comment:
For the internal would it make sense to say using `@internal` for that? Or are there other ways to tell Typedoc some types are internal?